core/src/v2/models.rs

```diff
@@ -332,6 +332,28 @@ pub struct Items {
     pub collection_format: Option<CollectionFormat>,
     #[serde(default, rename = "enum", skip_serializing_if = "Vec::is_empty")]
     pub enum_: Vec<serde_json::Value>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub maximum: Option<f32>,
+    #[serde(rename = "exclusiveMaximum", skip_serializing_if = "Option::is_none")]
+    pub exclusive_maximum: Option<bool>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub minimum: Option<f32>,
+    #[serde(rename = "exclusiveMinimum", skip_serializing_if = "Option::is_none")]
+    pub exclusive_minimum: Option<bool>,
+    #[serde(rename = "maxLength", skip_serializing_if = "Option::is_none")]
+    pub max_length: Option<i32>,
```

Comment:
Even though the spec says this is an integer, it also says that the integer must be greater than or equal to zero, in which case, we should go for the unsigned version (`u32`).